.changeset/real-penguins-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-typescript": minor
```

Comment:
```suggestion
"openapi-typescript": patch
```

I would consider this a `patch` because this doesn’t add new functionality; it fixes a bug, no?

Reply:
updated